Dockerfile

```diff
@@ -11,4 +11,4 @@ RUN python3 setup.py install
 COPY entrypoint.sh /
 RUN chmod +x /entrypoint.sh
 ENTRYPOINT ["/entrypoint.sh"]
-CMD ["-r", "--config-file", "config.ini"]
+CMD ["-r", "--config-file", "/etc/tilty/config.ini"]
```

Comment:
i think if we do this we should add a `VOLUME` specifier for `/etc/tilty` and update the docs to show `-v`

Reply:
```
VOLUME "/etc/tilty"
CMD ["-r", "--config-file", "/etc/tilty/config.ini"]
```

then maybe add to run: 

```
$ mkdir config
$ cat <<EOF >config/config.ini
[sqlite]
file = /etc/tilty/tilt.sqlite
EOF
$ docker run -it -v $(pwd)/config:/etc/tilty myoung34/tilty:latest
```
```